### ISSUE TITLE
Highlight the row which the mouse is hovering at

### ISSUE THREAD

**Issue by hopeng** [daniel-nagy/md-data-table#357]
Hi I don't seem to see this option. The row is highlighted on hover when table is enabled for selection. But I want this behavior for non selectable table as well, it's very useful when looking at wide tables. Is it possible? Thanks

**Comment by ducaale** [comment]
you can add this to your css 
`tbody > .md-row:hover {
  background-color: #EEEEEE;
}`

**Comment by hopeng** [comment]
Thanks! it works perfectly

**Comment by garretwilson** [comment]
Yeah, this was stumping me as well. Yes, the provided solution works perfectly. But if it's so simple, couldn't we have a directive such as `md-row-hover` to add to the table so that it does this automatically? Or better yet... why doesn't it just do it automatically anyway? Just curious --- it's not a huge thing.

**Comment by shennero** [comment]
Is there a way to suppress the row highlight on mouse hover even when row selection is turned on?
